core/storage/user/gae_models.py

```diff
@@ -164,13 +164,33 @@ def export_data(user_id):
             'role': user.role,
             'username': user.username,
             'normalized_username': user.normalized_username,
-            'last_agreed_to_terms': user.last_agreed_to_terms,
+            'last_agreed_to_terms': (
+                python_utils.convert_to_bytes(user.last_agreed_to_terms)
+                if user.last_agreed_to_terms
+                else None
+            ),
             'last_started_state_editor_tutorial': (
-                user.last_started_state_editor_tutorial),
+                python_utils.convert_to_bytes(
+                    user.last_started_state_editor_tutorial)
+                if user.last_started_state_editor_tutorial
+                else None
+            ),
             'last_started_state_translation_tutorial': (
-                user.last_started_state_translation_tutorial),
-            'last_logged_in': user.last_logged_in,
-            'last_edited_an_exploration': user.last_edited_an_exploration,
+                python_utils.convert_to_bytes(
+                    user.last_started_state_translation_tutorial)
```

Comment:
For datetimes, probably better to use strftime() rather than coercing to a string.

Reply:
Resolved